torch/onnx/symbolic_opset9.py

```diff
@@ -722,16 +728,44 @@ def prelu(g, self, weight):
 def silu(g, input):
     return g.op("Mul", input, g.op("Sigmoid", input))
 
+
 def mish(g, input):
     return g.op("Mul", input, g.op("Tanh", g.op("Softplus", input)))
 
+
+def op_with_optional_float_cast(g, op_name, self, *args, other_operands=None, opset_before=None, target_float_t="Float", **kwargs):
```

Comment:
Can we just use `args` instead of `other_operands`?
That is, are there cases where we don't want to cast all the op inputs?

Reply:
Good point. I just made the func signature compatible with previous usage.